spec/views/users_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+RSpec.describe 'User index Page', type: :feature do
+  let(:user1) do
+    User.create(
+      name: 'Tom',
+      photo: 'jane.jpg',
+      bio: 'sam is a 21 year old footballer from Ghana',
+      posts_counter: 2
+    )
+  end
+
+  let(:user2) do
+    User.create(
+      name: 'John Doe',
+      photo: 'john.jpg',
+      bio: 'john is a 21 year old footballer from Ghana',
+      posts_counter: 5
+    )
+  end
+
+  it 'displays users information' do
+    visit users_path
+
+    expect(page).to have_content(user1.name)
+    expect(page).to have_content(user2.name)
```

Comment:
- [ ] This test is failing while I run tests, Please make sure that all tests are passing.

![image](https://github.com/oguntayomicheal/Blogs-App/assets/68796232/401f5f81-0ab9-4de0-90db-006193670106)